docker/nginx-default.conf

```diff
@@ -0,0 +1,25 @@
+upstream mviewerstudio {
+   server mviewerstudio:8000;
+}
+
+upstream mviewer {
+   server mviewer:80;
+}
+
+server {
+    listen       8080;
+    server_name  localhost;
+
+    location /mviewer {
+        proxy_pass http://mviewer/;
+    }
+    location / {
+        proxy_pass http://mviewerstudio;
+
+        # Set accordingly to fake being behind geOrchestra's security-proxy
+        proxy_set_header sec-username "testadmin";
+        proxy_set_header sec-roles "ROLE_ADMINISTRATOR;ROLE_MVIEWER_ADMIN";
+        proxy_set_header sec-org "geobretagne";
```

Comment:
I think it can run without this. You just save stuff as "anonymous". Maybe comment it to let people use it if they want to fake georchestra proxy.. Since some user don't use mviewer in a georchestra context.

Reply:
of course this was intended for debugging purposes, I'll comment it.